<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FusionCore</class>
 <widget class="QDialog" name="FusionCore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="BoronInjector">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
Border-radius: 12px;
</string>
   </property>
   <property name="text">
    <string>Boron Injector</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AcceleratorField">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
Border-radius: 12px;</string>
   </property>
   <property name="text">
    <string>Accelerator Field</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Dc2AcConverter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
Border-radius: 12px;</string>
   </property>
   <property name="text">
    <string>DC to AC Conveter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CollectorField">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
Border-radius: 12px;</string>
   </property>
   <property name="text">
    <string>Collector Field</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AlphaPlasmaField">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
Border-radius: 12px;
</string>
   </property>
   <property name="text">
    <string>Alpha  Plasma Field</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BoronPlasmaField">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
Border-radius: 12px;</string>
   </property>
   <property name="text">
    <string>Boron Plasma Field</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AlphaInjector">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
Border-radius: 12px;</string>
   </property>
   <property name="text">
    <string>Alpha Injector</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Done">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Type">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Done</sender>
   <signal>clicked()</signal>
   <receiver>FusionCore</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BoronInjector</sender>
   <signal>clicked()</signal>
   <receiver>FusionCore</receiver>
   <slot>BoronInjection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AlphaInjector</sender>
   <signal>clicked()</signal>
   <receiver>FusionCore</receiver>
   <slot>AlphaInjection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BoronPlasmaField</sender>
   <signal>clicked()</signal>
   <receiver>FusionCore</receiver>
   <slot>BoronPlasmaField()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AcceleratorField</sender>
   <signal>clicked()</signal>
   <receiver>FusionCore</receiver>
   <slot>AcceleratorField()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CollectorField</sender>
   <signal>clicked()</signal>
   <receiver>FusionCore</receiver>
   <slot>CollectorField()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Dc2AcConverter</sender>
   <signal>clicked()</signal>
   <receiver>FusionCore</receiver>
   <slot>Dc2AcConverter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AlphaPlasmaField</sender>
   <signal>clicked()</signal>
   <receiver>FusionCore</receiver>
   <slot>AlphaPlasmaField()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BoronInjection()</slot>
  <slot>AlphaInjection()</slot>
  <slot>BoronPlasmaField()</slot>
  <slot>AlphaPlasmaField()</slot>
  <slot>AcceleratorField()</slot>
  <slot>CollectorField()</slot>
  <slot>Dc2AcConverter()</slot>
 </slots>
</ui>
